frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostsTable.tsx

```diff
@@ -93,7 +93,7 @@ const HostsTableRow: RowFunction<BareMetalHostBundle> = ({
         />
       </TableData>
       <TableData className={tableColumnClasses.status}>
-        <BareMetalHostStatus {...status} />
+        <BareMetalHostStatus {...status} nodeMaintenance={nodeMaintenance} host={host} />
```

Comment:
nodeMaintenance and host are already included in status object, so no need to pass them explicitly here.